### PR TITLE
Contributors guide

### PR DESCRIPTION
**Pull request by lukaszlenart**
This PR moves the Contributors Guide from Confluence into Markdown

**NOTE** Please squash this PR

### CODE REVIEW

File: source/contributors/building-with-maven.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+> Sometimes, licensing restrictions prevent Maven from downloading all the JARs that a build might need. For example, 
+> JavaMail and Activation, can only be downloaded from Sun. When this happens, Maven will display a helpful message 
+> that explains how to install these JARs manually. After downloading the required JAR, follow the instructions 
+> to install it to the your local repository. Once installed, the JAR is available to all your Maven builds, not just Struts.
+
+**Initial Build Successful**
+
+```
+[INFO] Reactor Summary:
```
Comment (JCgH4164838Gh792C124B5) [inline source/contributors/building-with-maven.md:55]: Similar to the above, the `Reactor Summary` here appears to be for the 2.0.x line.  Maybe using a more recent 2.5.x build's summary output could be used instead ?

File: source/contributors/building-with-maven.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+```
+mvn
+```
+
+That's it! Maven will download any dependencies the build needs; run all unit tests; package up the JARs; then install th
+e new JARs locally. For your convenience, copies of the JARs can be found in the _target_ directories of each module. 
+For example, after the build, the main JAR can found at 
+
+```
+core/target/struts2-core-2.0-SNAPSHOT.jar
```
Comment (JCgH4164838Gh792C124B5) [inline source/contributors/building-with-maven.md:44]: The reference is to the original Struts 2.0 release, could be updated to a newer one like `struts2-core-2.5.24-SNAPSHOT.jar`.  (meant to have been part of the review ... oops).

**Comment by lukaszlenart** [comment]
Sorry, this is still work-in-progress and I just opened the PR to test migration to new CI server :)

**Comment by yasserzamani** [comment]
LGTM :+1: thanks!

**Comment by lukaszlenart** [comment]
@apache/struts-committers this is ready

**Pull request merged**